masonry/src/event.rs

```diff
@@ -50,6 +50,84 @@ pub enum PointerButton {
     Other,
 }
 
+/// A set of [`PointerButton`]s.
```

Comment:
We should point to the web spec for their equivalent

Reply:
Maybe! There's some interesting differences between what we do for button / buttons and what the mouse event and pointer events specs say.

https://www.w3.org/TR/pointerevents3/#button-states

Reply:
(I think we should aim to move more towards pointer events and a cleaner model over time.)